lib/pages/stats/activity-page.js

```diff
@@ -18,7 +18,7 @@ export default class ActivityPage extends AsyncBaseContainer {
 				.isElementPresent(
 					driver,
 					By.xpath(
-						`//div[@class='activity-log-item__description-content']//a//em[text()='${ postTitle }']`
+						`//div[@class='activity-log-item__description-content']//a[text()='${ postTitle }']`
 					)
 				)
 				.then( present => {
```

Comment:
Should we consider and put async instead `.then` in this method?

Reply:
Good idea - fixed 👍